### PR TITLE
data_manager_manual: fix error on 18.01

### PR DESCRIPTION
**Pull request by abretaud**
Just a little fix for the manual data manager due to a class name change in 18.01 (https://github.com/galaxyproject/galaxy/commit/7df9dce63d635d66a0a9ce5045135ab49b7ef506)
With this patch + https://github.com/galaxyproject/galaxy/pull/5922 the data manager seems to work fine in the 18.01 docker image

### CODE REVIEW

**Comment by mvdbeek** [comment]
@galaxybot test this

File: data_managers/data_manager_manual/data_manager/data_manager_manual.py [hunk 1]
```diff
@@ -52,8 +52,11 @@ def exec_before_job( app, inp_data, out_data, param_dict, tool=None, **kwd ):
                 data_table_filename = data_table.get_filename_for_source( data_manager, None )
                 if not data_table_filename:
                     if tdtm is None:
-                        from tool_shed.tools import data_table_manager
-                        tdtm = data_table_manager.ToolDataTableManager( app )
+                        try:
+                            from tool_shed.tools.data_table_manager import ToolDataTableManager as ShedToolDataTableManager  # For Galaxy < 18.01
```
Comment (bgruening) [inline data_managers/data_manager_manual/data_manager/data_manager_manual.py]: uh, @jmchilton should we also fix this in Galaxy?

Comment (mvdbeek) [reply data_managers/data_manager_manual/data_manager/data_manager_manual.py]: Do data managers actually use this ? In that case we should rename things, yes.

Comment (mvdbeek) [reply data_managers/data_manager_manual/data_manager/data_manager_manual.py]: Urgs, the rsync manager does :(

Comment (abretaud) [reply data_managers/data_manager_manual/data_manager/data_manager_manual.py]: On IUC it's used in https://github.com/galaxyproject/tools-iuc/blob/master/data_managers/data_manager_rsync_g2/data_manager/data_manager_rsync.py#L85 too, no idea for other DMs

Comment (abretaud) [reply data_managers/data_manager_manual/data_manager/data_manager_manual.py]: I can patch it too if it's the only one

Comment (mvdbeek) [reply data_managers/data_manager_manual/data_manager/data_manager_manual.py]: We can't make that assumption, IMO, we don't know who relies on this.
It's easy though to fix this in galaxy.

**Comment by bgruening** [comment]
@abretaud can you try latest 18.01?

**Comment by abretaud** [comment]
https://github.com/galaxyproject/galaxy/pull/5924 Tested & Approved! Thanks
Could you trigger a new build of the 18.01 tag on https://quay.io/repository/bgruening/galaxy?

**Pull request closed without merging**